ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/adapter/util/SampleData.java

```diff
@@ -472,9 +470,9 @@ public static CCDCase getCCDLegalCase() {
             .amountType(RANGE)
             .amountLowerValue("5000")
             .amountHigherValue("50000")
-            .housingDisrepairCostOfRepairDamages(MORE_THAN_THOUSAND_POUNDS.name())
-            .housingDisrepairOtherDamages(THOUSAND_POUNDS_OR_LESS.name())
-            .personalInjuryGeneralDamages(MORE_THAN_THOUSAND_POUNDS.name())
+            .housingDisrepairCostOfRepairDamages("MORE_THAN_THOUSAND_POUNDS")
```

Comment:
Prefer not to use strings here, would be helpful in refactoring if using enum constants here. You may revert this change.